src/components/Footer/Footer.js

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import Styled from 'styled-components';
+// import { ReactComponent as FooterLogo } from '../Header/logo.svg';
+import { ReactComponent as FooterLogo } from './sign.svg';
+
+const StyledFooter = Styled.footer`
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  width: 100%;
+  height: 100px;
+  padding: 0 250px;
+  box-sizing: border-box;
+
+  a.footer-link {
+    font-size: 13px;
+    letter-spacing: .6px;
+    color: #636363;
+  }
+
+  .footer-logo {
+    margin-left: 55px;
+  }
+`;
+
+
+export default function Footer() {
+  return (
+    <StyledFooter>
+      <a
+        className="footer-link"
```

Comment:
Don't use global classes. Rather create a separate styled component `FooterLink`. Same goes for the links below